backend/backend/urls.py

```diff
@@ -118,10 +118,17 @@
 # Clone urls
 try:
 
-    import pluggable_apps.clone.urls  # noqa # pylint: disable=unused-import
-
     urlpatterns += [
         path("", include("pluggable_apps.clone.urls")),
     ]
 except ImportError:
     pass
+
+try:
+    import pluggable_apps.apps.table_settings  # noqa # pylint: disable=unused-import
+
+    urlpatterns += [
+        path("", include("pluggable_apps.apps.table_settings.urls")),
+    ]
+except ImportError:
+    pass
```

Comment:
@harini-venkataramanShould we add an info log here to understand if there was any prob in loading libs for easy debugging. IN the past, i have heard that there were some hiccups in understanding the code flow here from @tahierhussain too...